projects/netgrif-components-core/src/lib/task-content/services/field-converter.service.ts

```
@@ -79,14 +79,14 @@ export class FieldConverterService {
             case FieldTypeResource.USER:
                 let user;
                 if (item.value) {
-                    user = new UserValue(item.value.id, item.value.firstName, item.value.lastName, item.value.email);
+                    user = new UserValue(item.value.id, item.value.realmId, item.value.firstName, item.value.lastName, item.value.username);
                 }
                 return new UserField(item.stringId, item.name, item.behavior, user,
                     item.roles, item.placeholder, item.description, item.layout, item.validations, item.component, item.parentTaskId);
             case FieldTypeResource.USER_LIST:
                 let userListValue = new UserListValue(new Map<string, UserValue>());
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)